index.bs

```diff
@@ -213,10 +212,7 @@ spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180
   specified in RFC5234. [[!ABNF]]
 
   <a href="https://tools.ietf.org/html/rfc5234#appendix-B.1">Appendix B.1</a> of
-  [[!ABNF]] defines <a>VCHAR</a> (printing characters).
-
-  <a>WSP</a> (white space) characters are defined in <a href="http://www.w3.org/TR/html5/infrastructure.html#space-character">Section 2.4.1 Common parser idioms</a> of the HTML 5 specification as
-  <code>White_Space characters</code>. [[!HTML5]]
+  [[!ABNF]] defines <a>VCHAR</a> (printing characters) and <a>WSP</a> (white space).
```

Comment:
Please check.

Reply:
So the previous text makes no sense, but I wonder if it was meant to match https://infra.spec.whatwg.org/#ascii-whitespace? That would be `LWSP / %x0C`.

This also seems like a problem with the parser. It references https://infra.spec.whatwg.org/#strictly-split but doesn't define spaces. I suspect it meant to reference https://infra.spec.whatwg.org/#split-on-ascii-whitespace?

Are there tests for this?

Reply:
As you said, [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace) seems to be equivalent to `LWSP`. And the implementation, Chromium, use ASCIISpace defined as follows. 

```
inline bool IsASCIISpace(CharType c) {
  return c <= ' ' && (c == ' ' || (c <= 0xD && c >= 0x9));
}
```

In this situation, as you said, I think it would be better to refer to [split-on-ascii-whitespace](https://infra.spec.whatwg.org/#split-on-ascii-whitespace) rather than [strictly-split](https://infra.spec.whatwg.org/#strictly-split) for the parser. If so, it seems fitting to use `LWSP`, which refers to [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace), instead of `WSP`. The ABNF in Section 3.5 will also need to be modified accordingly. Does it make sense?

Reply:
So Chrome's definition also counts U+000B as whitespace if I'm reading that correctly. That seems wrong.

And note that LWSP excludes U+000C (which is why I added that above).

I think we should use ASCII whitespace here and file a bug on Chrome for them including U+000B. And use `LWSP / %x0C` in ABNF.